_monitoring-plugins/alerting/monitors.md

```diff
@@ -156,17 +156,12 @@ Whereas per-query monitors run your specified query and then check whether the q
      }
      ```
 
-     "Start" and "end" refer to the interval at which the monitor runs. See [Available variables](#available-variables).
+    "Start" and "end" refer to the interval at which the monitor runs. See [Available variables](#available-variables).
 
-
-1. Choose a frequency and timezone for your monitor. Note that you can only pick a timezone if you choose Daily, Weekly, Monthly, or [custom cron expression]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/cron/) for frequency.
-
-1. Choose one or more indices. You can also use `*` as a wildcard to specify an index pattern.
+    To define a monitor visually, choose **Visual editor**. Then choose a source index, a timeframe, an aggregation (for example, `count()` or `average()`), a data filter if you want to monitor a subset of your source index, and a group-by field if you want to include an aggregation field in your query. Visual definition works well for most monitors.
```

Comment:
Maybe a call for @aetter but I think you can still leave in the "At least one group-by field is required if you are creating a per bucket monitor." as the second to last sentence since it is a useful bit of info. Although, this is required from the UI and we mention that there, so the user will still find out either way.

Reply:
Oops, I didn't mean to take that out. I'll add it back in.